src/main/java/ch/njol/skript/expressions/ExprColorOf.java

```diff
@@ -57,29 +59,29 @@
 public class ExprColorOf extends PropertyExpression<Object, Color> {
```

Comment:
Could you update the description to include all the new changes? And examples as well